front-end/src/components/Button/props.js

```diff
@@ -0,0 +1,18 @@
+import PropTypes from 'prop-types';
+
+export const propTypes = {
+  children: PropTypes.node,
```

Comment:
Re-arrange the proptypes. For example (strings together)

Reply:
The proptypes are now grouped together by type.